package.js

```diff
@@ -8,6 +8,7 @@ auto.versionFile()
 content.description = 'Listens on editable elements and calls back on significant changes'
 content.main = 'lib/Edited/index.js'
 content.dependencies = {
+  combokeys: '^2.4.3',
```

Comment:
`combokeys` is a [valid identifier name](https://mathiasbynens.be/notes/javascript-identifiers) so it doesn't require quotes.

`add-event-handler` and `dom-events` do, because of the hyphens.

`keycode` is needlessly a string.